### PR TITLE
[DR-2743] Added new tabs to the navbar

### PR DESCRIPTION
**Pull request by lu-c**
- Added new tabs to the navbar
- Moved routes and tabs to a javascript object (to allow the next bullet to happen)
- Filter tabs to show by what routes exist

BEFORE
Screen Shot 2022-09-15 at 11 52 40 AM

AFTER
Screen Shot 2022-09-15 at 12 26 06 PM
Screen Shot 2022-09-15 at 12 26 12 PM

### CODE REVIEW

File: src/routes/Private.jsx [hunk 1]
```diff
@@ -61,6 +63,21 @@ const styles = (theme) => ({
...
 
+const tabsConfig = [
+  { label: 'Datasets', path: '/datasets' },
+  { label: 'Snapshots', path: '/snapshots' },
+  { label: 'Jobs', path: '/jobs' },
+  { label: 'Ingest Data', path: '/ingestdata' },
+];
+
+const routes = [
+  { path: '/datasets', component: HomeView },
```
Comment (lu-c) [inline src/routes/Private.jsx:73]: I mean, this could be overkill. I'm happy to revert this back to what it was before!

Comment (okotsopoulos) [reply src/routes/Private.jsx:73]: No, I like this!  I see that you get some reuse from it downstream, and even if you didn't I like how it's clear that altering the routes would take place here.

Comment (otchet-broad) [reply src/routes/Private.jsx:73]: I agree.  This is a nice change.

**Review by otchet-broad — APPROVED** [review]
Looks good!

**Review by okotsopoulos — APPROVED** [review]
Whoops, forgot to approve earlier :) this looks great!

**Pull request merged**